app/src/main/java/com/example/activeamigo/Algorithm.java

```diff
@@ -120,16 +122,18 @@ static List<HashMap<String,Object>> findSimilarSchedulesForUser(String user1, Li
             String userGender = document.getString("gender");
 
             if (userId.equals(user1)) continue; // Skip user1
-
             if (user1Gender != null) {
+                if(!user1Gender.equals("Select Gender"))

```

Comment:
Please add comment here to explain, im not sure what this if statement is doing since it looks like it is an an empty. If this is a nested if statement, please indent for styling consistency

Reply:
Its a nested if statement to not skip a user in case of default filter
I'll add the indentation